packages/components/src/components/DistributionChart.tsx

```diff
@@ -72,6 +76,21 @@ export const DistributionChart: React.FC<DistributionChartProps> = (props) => {
       );
     }
 
+    // if this is a sample set, include the samples
+    const samples: number[] = [];
+    const sampleSets = plot?.distributions.filter(
+      (dist) => dist.distribution.tag === SqDistributionTag.SampleSet
+    );
+    if (sampleSets.length) {
+      for (const { distribution } of sampleSets) {
+        if (distribution.tag === SqDistributionTag.SampleSet) {
+          // this conditional must be duplicated to please typescript, more elegant solution probably exists
+          samples.push(...distribution.value());
+          // shapes.value[0].samples.push(...distribution.value());
+        }
+      }
+    }
```

Comment:
Note that you could build samples in a single pass now, and avoid double `SqDistributionTag.SampleSet` check.

Also, please clean up the old commented out versions of code (here and above where we map `plot.distributions`).